personal/privacyidea/class_PrivacyIdeaUtils.inc

```diff
@@ -224,6 +230,10 @@ class PrivacyIdeaUtils implements PILog
         assert(!empty($tokenCode));
         assert($tokenType == "paper" || $tokenType == "totp");
 
+        if (!$this->authenticate()) {
+            return array();
+        }
+
```

Comment:
dito here

Reply:
The above is equal to the existing failure mode of `piSendRequest` and handled properly.